catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -126,7 +136,30 @@ private CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
                 filterBuilder);
 
         Stopwatch stopwatch = Stopwatch.createStarted();
-        QueryResponse response = catalogFramework.query(request);
+        QueryResponse response = null;
+
+        if (configuration.isDisableLocalCatalog()) {
```

Comment:
I didn't want to make any assumption since the request being sent can support multiple source ids, I wanted to make sure that the code would perform correctly.